mobile-bare/README.md

```diff
@@ -0,0 +1,78 @@
+Lightning mobile app
+==========
+
+## Getting started
+
+This project was bootstrapped using [Expo CLI as a bare project](https://blog.expo.io/you-can-now-use-expo-apis-in-any-react-native-app-7c3a93041331). You should only need `react-native-cli` for development not `expo-cli`. See the [React Native CLI Getting Started Guide](https://facebook.github.io/react-native/docs/getting-started.html)
+
+## Building lnd
+
+### Build dependencies
+
+* node 10 LTS
+* react-native-cli (npm install -g react-native-cli)
+* cocoapods 1.7.2
+* go 1.11.6
+* [gomobile](https://github.com/golang/go/wiki/Mobile)
+* [lnd mobile build tools](https://github.com/lightninglabs/lnd/tree/mobile-build-tool)
+
+### Building
+
+```
+go get golang.org/x/mobile/cmd/gomobile
+cd lnd
+make clean
+gomobile init
+make mobile
+```
+
+Next copy the lnd mobile binaries to this project
+
+1. `lnd/mobile/build/ios/Lndmobile.framework` -> `ios/lightning`
+2. `lnd/mobile/build/android/Lndmobile.aar` -> `android/Lndmobile`
+
+## Running the app
+
+### Install the dependencies
+
+```
+npm install
+cd ios
+pod install
```

Comment:
Ran into [this](https://github.com/CocoaPods/CocoaPods/issues/8138) problem. I was able to get it to work by downgrading to cocoapods 1.5.3, but the downgrade caused problems for `npm run ios` and the Xcode build.

Reply:
> I was able to get it to work by downgrading to cocoapods 1.5.3

As far as I understand that was necessary only for the Expo 32 build. The bare builds support the new cocoapods.